webcomponents/tool/src/main/frontend/js/rubrics/sakai-rubric-summary.js

```diff
@@ -0,0 +1,363 @@
+                            <div class="panel panel-default">
+                                <div class="panel-heading">
+                                    <h4 class="panel-title">
+                                        <a class="collapse-toggle collapsed" data-toggle="collapse" href="#collapse${c.id}">${c.title}</a>
+                                    </h4>
+                                </div>
+                                <div id="collapse${c.id}" class="panel-collapse collapse">
+                                    <div class="panel-body">
+                                        <div class="table-responsive">
+                                <table class="rubrics-summary-table table table-bordered table-condensed">
+                                    <tr>
+                                        ${c.ratings.map((r) => html`
+                                            <th class="rubrics-summary-table-cell">
+                                                <div>${r.points} <sr-lang key="points">points</sr-lang></div>
+                                                <div class="summary-rating-name" title="${r.title}">${this.limitCharacters(r.title, 20)}</div>
+                                            </th>
+                                            ${this.association.parameters.fineTunePoints && this.getCustomCount(c.id, r.points)>0 ? html`
+                                                <th class="rubrics-summary-table-cell"><sr-lang key="adjusted_score">adjustedscore</sr-lang></th>
+                                            ` : html``}
+                                        `)}
+                                        <th style="display:none" class="rubrics-summary-table-cell rubrics-summary-average-cell"><sr-lang key="average">average</sr-lang></th>
+                                        <th style="display:none" class="rubrics-summary-table-cell "><sr-lang key="median">median</sr-lang></th>
+                                        <th style="display:none" class="rubrics-summary-table-cell "><sr-lang key="stdev">stdev</sr-lang></th>
+                                    </tr>
+                                    <tr>
+                                        ${c.ratings.map((r) => html`
+                                            <td class="points-${r.points} rubrics-summary-table-cell pointCell-${c.id}" >${this.getACount(c.id, r.id)}</td>
+                                            ${this.association.parameters.fineTunePoints && this.getCustomCount(c.id, r.points)>0 ? html`
+                                                <td class="rubrics-summary-table-cell">${this.getCustomCount(c.id, r.points)}</td>
+                                            ` : html``}
+                                        `)}
+                                        <td style="display:none" class="rubrics-summary-table-cell rubrics-summary-average-cell">${this.getPointsAverage(c.id)}</td>
```

Comment:
I prefer not using inline styles but I'm still skilling up on web components, so if it's best practice there don't worry about removing them.